ruby_exercises_18_01_26/ruby_exercises_q3.rb

```diff
@@ -0,0 +1,12 @@
+def isprime? num
+  divisor = num - 1
+  if num < 2
+    return false
+  else
```

Comment:
This works as expected but is fairly slow for large prime numbers. How might you make it more efficient?
Prime numbers after 2 are never even.